src/templates/blog-home.tsx

```diff
@@ -0,0 +1,45 @@
+import { graphql } from 'gatsby'
```

Comment:
is there analog for doc page? should it be unified - the way we process pages?

Reply:
I refactored `doc.js` to TS, and renamed it to `doc-home.ts` to keep consistensy with other templates. Is that what you meant?

Reply:
👍 yes, that was a comment about some consistency how we build and process stuff